app/pages/lab/project.cjsx

```diff
@@ -252,7 +252,7 @@ module.exports = React.createClass
           <p className="form-help">Loading project</p>
         </div>
       } then={([project, owners]) =>
-        if @props.user in owners
+        if @props.user in owners or @props.user.admin
```

Comment:
Use the function at `lib/is-admin` to check if the user is properly in admin mode.